notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -220,6 +220,56 @@ father and son [Sherman Klump had a heart to heart talk with his father Cletus a
 giant hamster []
 
 
+movie: Hollow Man (2000)
+========================
+
+:: Title
+Hollow Man
+
+:: Description
+Hollow Man is a 2000 science fiction horror film directed by Paul Verhoeven
+and starring Kevin Bacon, Elisabeth Shue and Josh Brolin.
+
+Synopsis: A scientist volunteers to be the first human test subject for a
+serum that renders the user invisible. When his fellow scientists are unable
+to restore him back to normal, he eventually becomes violently insane and goes
+on a murderous rampage.
+
+:: Date
+2000-08-04
+
+:: References
+https://en.wikipedia.org/wiki/Hollow_Man
+
+:: Ratings
+3 <paul>
+
+:: Choice Themes
+personal invisibility cloak [A main novelty of the film involves the scientist Sebastian Caine injecting himself with an experimental serum that renders its subject invisible and its unfortunate aftermath.]
```

Comment:
Brainstorming: I guess the theme we are really interested in could be called "invisibility effect" with children for "personal", "planetary" or whatever variants. Then we'd not have to worry about if it's technology, a serum, a potion, wizard magic or divine miracle etc etc.

Reply:
Now that's an idea with some potential. Scratching my head over where **invisibility effect** would in the **speculative fiction theme** hierarchy .

Reply:
I think we'd need a new category "speculative effect". Perhaps abilities are all effects that can be commanded at will. Some technologies and superstition themes may imply certain effects. For example we can keep the invisibility cloak if we like and parent it to the new effect.

Reply:
I like! It's on the meeting agenda.